ee/clickhouse/models/event.py

```diff
@@ -88,10 +88,22 @@ def bulk_create_events(events: List[Dict[str, Any]]):
             elements_chain = elements_to_string(elements=event.get("elements"))  # type: ignore
 
         inserts.append(
-            "(%(uuid_{i})s, %(event_{i})s, %(properties_{i})s, %(timestamp_{i})s, %(team_id_{i})s, %(distinct_id_{i})s, %(elements_chain_{i})s, %(created_at_{i})s, now(), 0)".format(
+            "(%(uuid_{i})s, %(event_{i})s, %(properties_{i})s, %(timestamp_{i})s, %(team_id_{i})s, %(distinct_id_{i})s, %(elements_chain_{i})s, %(person_id_{i})s, %(person_properties_{i})s, %(group0_properties_{i})s, %(group1_properties_{i})s, %(group2_properties_{i})s, %(group3_properties_{i})s, %(group4_properties_{i})s, %(created_at_{i})s, now(), 0)".format(
                 i=index
             )
         )
+
+        #  use person properties mapping to populate person properties in given event
+        if person_mapping and person_mapping.get(event["distinct_id"]):
```

Comment:
Going to need to handle group properties in some way like this too

Reply:
Aye, leaving it for a PR where groups get tackled, as that provides an easy way to test the changes are sound.

Right now, since there's no code actually using the instance setting, the new CI stuff still runs the old code, with extra event properties populated.